packages/xo-server-test/.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 2021,  // Compatible avec ES12+
+    "sourceType": "module"
+  },
+  "overrides": [
+    {
+      "files": ["*.js", "*.{integ,test}.{js,jsx}"],
+      "rules": {
+        "new-cap": ["warn"],  // Fix possible typo in rule
+        "no-console": "warn",
+        "n/no-unsupported-features/es-builtins": ["warn"],
+        "n/no-unsupported-features/es-syntax": ["warn"],
+        "n/no-missing-import": ["warn"],
+        "no-unused-vars": ["warn"]
+      }
+    }
+  ]
+}
```

Comment:
Do we still need this file , or do the global eslintrc do the job ?

Reply:
There are errors on imports without the "sourceType: module", and the overrides are here only to warn about problems that must be fixed in other tests.

Reply:
Done.
I removed the eslintrc.

On 
`const xo = new Xo.default({ url })`
Doesn't respect the eslint rule : A constructor name should not start with a lowercase letter. (new-cap)

I changed it to :
```
const XoConnection = Xo.default
  const xo = new XoConnection({ url })

```